### PR TITLE
Added bottomPanel to defaultFrontstage

### PR DESCRIPTION
**Pull request by nick4598**
In order to add the modelAlignmentTable to the bottom of the frontstage a bottomPanel is necessary. This is because the UiItemsProvider only adds widgets to panels that exist in the frontstage. 
Without this bottomPanel, the modelAlignmentTable widget gets ignored.

### CODE REVIEW

File: packages/modules/itwin-viewer-react/src/components/app-ui/frontstages/DefaultFrontstage.tsx [hunk 1]
```diff
@@ -2,6 +2,10 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
+/*---------------------------------------------------------------------------------------------
```
Comment (aruniverse) [inline packages/modules/itwin-viewer-react/src/components/app-ui/frontstages/DefaultFrontstage.tsx:5]: remove duplicate header

**Review by aruniverse — COMMENTED** [review]
Also need a rush change log

File: packages/modules/itwin-viewer-react/src/components/app-ui/frontstages/DefaultFrontstage.tsx (hunk 1 above)
Comment (nick4598) [reply packages/modules/itwin-viewer-react/src/components/app-ui/frontstages/DefaultFrontstage.tsx]: removed header and ran rush change

**Review by kckst8 — APPROVED** [review]

**Review by kckst8 — CHANGES_REQUESTED** [review]
Can you also add the leftPanel while you're at this?

**Comment by nick4598** [comment]
I added left panel as well, just picked 1,4,7 for allowedZones as those are the left locations (TopLeft, CenterLeft, BottomLeft)

**Review by kckst8 — APPROVED** [review]
Thanks!

File: packages/modules/itwin-viewer-react/src/components/app-ui/frontstages/DefaultFrontstage.tsx [hunk 2]
```diff
@@ -192,6 +192,7 @@ export class DefaultFrontstage extends FrontstageProvider {
         }
         rightPanel={<StagePanel allowedZones={[6, 9]} />}
         bottomPanel={<StagePanel allowedZones={[7, 8, 9]} />}
+        leftPanel={<StagePanel allowedZones={[1, 4, 7]} />}
```
Comment (aruniverse) [inline packages/modules/itwin-viewer-react/src/components/app-ui/frontstages/DefaultFrontstage.tsx:195]: idk if 1 should be an allowed zone, typically used for tools, but ill defer to Kevin

Comment (kckst8) [reply packages/modules/itwin-viewer-react/src/components/app-ui/frontstages/DefaultFrontstage.tsx:195]: the panel "should show up to the left of the tools if all are being utilized

**Pull request merged**